site/src/content/docs/ref/init-package.mdx

```diff
@@ -134,58 +141,166 @@ package:
 
-## The `zarf init` Lifecycle
-
-The `zarf init` lifecycle is _very similar_ to the [`zarf package deploy` lifecycle](/ref/deploy/) except that it sets up resources specific to Zarf such as the `zarf-state` and performs special actions such as the injection procedure.
-
-<Mermaid diagram={`
-graph TD
-    B1(load package archive)-->B2
-    B2(handle multipart package)-->B3
-    B3(extract archive to temp dir)-->B4
-    B4(validate package checksums and signature)-->B5
-    B5(filter components by architecture & OS)-->B6
-    B6(save SBOM files to current dir)-->B7
-    B7(handle deprecations and breaking changes)-->B9
-    B9(confirm package deploy):::prompt-->B10
-    B10(process deploy-time variables)-->B11
-    B11(prompt for missing variables)-->B12
-    B12(prompt to confirm components)-->B13
-    B13(prompt to choose components in '.group')-->B14
-
-    subgraph  ""
-    B52 --> |Yes|B14(deploy each component)-->B14
-    B14 --> B15{Component is \n zarf-seed-registry}
-    B15 --> |Yes|B51(initialize zarf-state secret):::action
-    B51 --> B52{External \n registry configured}
-    B52 --> |No|B53(run injection process):::action-->B16
-    B15 --> |No|B16(run each '.actions.onDeploy.before'):::action-->B16
-    B16 --> B17(copy '.files')-->B18
-    B18(load Zarf State)-->B19
-    B19(push '.images')-->B20
-    B20(push '.repos')-->B21
-    B21(process '.dataInjections')-->B22
-    B22(install '.charts')-->B23
-    B23(apply '.manifests')-->B24
-    B24(run each '.actions.onDeploy.after'):::action-->B24
-    B24-->B25{Success?}
-    B25-->|Yes|B26(run each\n'.actions.onDeploy.success'):::action-->B26
-    B25-->|No|B27(run each\n'.actions.onDeploy.failure'):::action-->B27-->B999
-
-    B999[Abort]:::fail
-    end
-
-    B26-->B28(print Zarf connect table)
-    B28-->B29(save package data to cluster)
-
-
-    classDef prompt fill:#4adede,color:#000000
-    classDef action fill:#bd93f9,color:#000000
-    classDef fail fill:#aa0000
-`}/>
+### `zarf-agent`
+
+{/* TODO: document and flesh out how the mutations operate for the agent */}
+
+The `zarf-agent` is a [Kubernetes Mutating Webhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) that intercepts requests to create resources and uses the `zarf-state` secret to mutate them to point to their air-gapped equivalents.
+
+The `zarf-agent` is responsible for modifying [Kubernetes PodSpec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) objects [Image](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container.Image) fields to point to the Zarf Registry.
+
+This allows the cluster to pull images from the Zarf Registry instead of the internet without having to modify the original image references.
+
+The `zarf-agent` also modifies [Flux GitRepository](https://fluxcd.io/docs/components/source/gitrepositories/) objects to point to the local Git Server.
```

Comment:
Might be good to include that we the agent also mutates ArgoCD `Application` and `Repository` objects

Reply:
I left a TODO to really flesh out how the mutations work, but I will add that info